src/stratum_server.cpp

```diff
@@ -1024,14 +1025,16 @@ void StratumServer::on_share_found(uv_work_t* req)
 			prev_time = server->m_lastSidechainShareFoundTime;
 			server->m_lastSidechainShareFoundTime = cur_time;
 		}
+		if (share->m_highEnoughDifficulty)
+		{
+			if (!pool->submit_sidechain_block(share->m_templateId, share->m_nonce, share->m_extraNonce)) {
```

Comment:
These two can be combined `if (share->m_highEnoughDifficulty && !pool->submit_sidechain_block...`

Reply:
will update, thanks for review

Reply:
combined the two ifs as recommended in https://github.com/SChernykh/p2pool/pull/374/commits/164e598677bdf7889328a5c766e81d50911abdee